read.go

```diff
@@ -110,12 +111,6 @@ func readPixelData(r dicomio.Reader, t tag.Tag, vr string, vl uint32, d *Dataset
 	if vl == tag.VLUndefinedLength {
 		var image PixelDataInfo
 		image.IsEncapsulated = true
-		// The first Item in PixelData is the basic offset table. Skip this for now.
-		// TODO: use basic offset table
-		_, _, err := readRawItem(r)
-		if err != nil {
-			return nil, err
-		}
```

Comment:
@suyashkumar I removed this code because some of the files I was testing the write functionality on did not use a basic offset table. By removing this code the offset tables were just read in as frames and then were rewritten byte for byte identical to the original file

Reply:
That sounds good. I'll revert the changes in this PR since there is no real impact on the resulting written DICOMs. I think the changes you proposed would be great to have in a follow up PR and I'd be happy to help out with the implementation